lib/astutils.cpp

```diff
@@ -568,7 +568,7 @@ static std::vector<const Token*> getParentMembers(const Token* tok)
         return {};
     if (!Token::simpleMatch(tok->astParent(), "."))
         return {tok};
-    const Token* parent = tok;
+    const Token* parent = tok->astParent();
```

Comment:
This wont have the same behavior.

Reply:
It does - the code below checks if `astParent()` is `.` and then jumps to the next `astParent()`. Because the check above already made that check once we skip the first.